src/status_im2/constants.cljs

```diff
@@ -331,3 +331,9 @@
 (def ^:const auth-method-biometric "biometric")
 (def ^:const auth-method-biometric-prepare "biometric-prepare")
 (def ^:const auth-method-none "none")
+
+(def ^:const initials-avatar-font-uppercase-ratio
```

Comment:
could you please elaborate why we need these 3 consts?

Reply:
We need to pass the absolute font file path to status-go so that it can render initials avatar with the specified font

ios and android use different font formats
the `:uppercase-ratio` is there because it's bit complex and heavy (I guess) to calculate this on each render, and I believe the font for initials avatar won't change too much.

I've put these 3 into one `constants/initials-avatar-font-conf` so that in case we need to change the font, we only have to change these 3 values in cljs.